builtin/logical/pki/path_config_ca.go

```diff
@@ -11,6 +11,12 @@ import (
 func pathConfigCA(b *backend) *framework.Path {
 	return &framework.Path{
 		Pattern: "config/ca",
+
+		DisplayAttrs: &framework.DisplayAttributes{
+			OperationPrefix: operationPrefixPKI,
+			OperationSuffix: "ca-config",
```

Comment:
Since the verb is not specified here, it will be implied from the operation. The generated operation ID is going to be `pki-write-ca-config`. Most of the other config endpoints have a read + write pair so I wanted to keep this consistent:

| Path | OperationIDs |
| ----| --------- |
| "/{pki_mount_path}/config/auto-tidy" |        pki-read-auto-tidy-config |
| | pki-write-auto-tidy-config |
| "/{pki_mount_path}/config/ca" |        pki-write-ca-config |
| "/{pki_mount_path}/config/cluster" |        pki-read-cluster-config |
| | pki-write-cluster-config |
| "/{pki_mount_path}/config/crl" |        pki-read-crl-config |
| | pki-write-crl-config |
| "/{pki_mount_path}/config/issuers" |        pki-read-issuers-config |
| | pki-write-issuers-config |
| "/{pki_mount_path}/config/keys" |        pki-read-keys-config |
| | pki-write-keys-config |
| "/{pki_mount_path}/config/urls" |        pki-read-url-config |
| | pki-write-url-config |

Reply:
Ah, I gotcha now :-)

Reply:
After some discussion for related PRs we decided to use "configure-x" pattern for all UPDATE configuration requests and "read-x-config" for GET requests. The new list is:

| Path | OperationIDs |
| ----| --------- |
| "/{pki_mount_path}/config/auto-tidy" | pki-read-auto-tidy-config |
| | pki-configure-auto-tidy |
| "/{pki_mount_path}/config/ca" | pki-configure-ca |
| "/{pki_mount_path}/config/cluster" | pki-read-cluster-config |
| | pki-configure-cluster |
| "/{pki_mount_path}/config/crl" | pki-read-crl-config |
| | pki-configure-crl |
| "/{pki_mount_path}/config/issuers" | pki-read-issuers-config |
| | pki-configure-issuers |
| "/{pki_mount_path}/config/keys" | pki-read-keys-config |
| | pki-configure-keys |
| "/{pki_mount_path}/config/urls" | pki-read-urls-config |
| | pki-configure-urls |